<launch>
    <node pkg="cmd_video" 
        type="rgb.py" 
        name="listen_rgb" 
        output="screen">
    <param name="rgb_stamp" type="str" value="$(arg rgb_stamp)"></param>
    <param name="rgb_id" type="int" value="$(arg idx_rgb)"></param>
    </node>

    <node pkg="cmd_video" 
        type="depth.py" 
        name="listen_depth" 
        output="screen">
    <param name="depth_stamp" type="str" value="$(arg depth_stamp)"></param>
    <param name="depth_id" type="int" value="$(arg idx_depth)"></param>
    </node>
</launch>
